docs/source/advancedcommands.rst

```diff
@@ -233,3 +233,24 @@ Example Usage
 The ``normsub`` argument specifies an expected value for your solution to normalize the different :math:`g_i` (you can also do this by hand). The feasibility of the problem should not depend on the normalization, but the spacing of the sweep will.
 
 The ``sweep`` argument specifies what points between 0 and 1 you wish to sample the weights at. If you want different resolutions or spacings for different weights, the ``sweeps`` argument accepts a list of sweep arrays.
+
+
+Debugging
+=========
+
+Unbounded variables
+-------------------
+In some cases a solver will return ``unknown`` when a free variable is
+unbounded by the model. ``gpkit.constraints.bounded.BoundedConstraintSet`` is a
+simple tool that attempts to detect unbounded variables by adding extremely
+large upper bounds and extremely small lower bounds to all variables in a ConstraintSet,
+resolving, and checking whether any variables slide off to the bounds.
```

Comment:
technically, the BoundedConstraintSet does not resolve; the above should say something more like "if a model involving a BoundedConstraintSet is resolved, warnings will be raised for any variables that slide off to the bounds."